### DISCUSSION TITLE
Constant DNS PTR Lookups for Private IP's

### DISCUSSION THREAD

**Discussion by dayton3375** [Security-Onion-Solutions/securityonion#8462] [Unsupported Versions]
Security Onion seems to have a problem for me where it sends out a lot of DNS PTR lookups for the following:
1. It's management IP such as "64.1.168.192.in-addr.arpa"
2. A repeated PTR lookup of "1.0.17.172.in-addr.arpa"

So basically every few seconds Security Onion will ask my router's DNS server a reverse lookup for it's own IP address and the private IP address of 172.17.0.1. Does anyone know why it's repeatedly looking up it's own private IP's? It ends up hitting my router's DNS around 30,000 times a day with these.

I'm running a Standalone Security Onion server in Hyper-V and it is a pretty standard setup. Over the past year I've re-installed it several times with the newest ISO and it has always had this problem even with the latest update (sudo soup).

**Comment by ddekam** [comment]
This was fixed for me in the newest version 2.4.10. Up until then I experienced the issue so this is the first update that fixed it.
